modules/installing-rhv-config-yaml.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_rhv/installing-rhv-custom.adoc
+// * installing/installing_rhv/installing-rhv-default.adoc
+
+[id="installing-rhv-installation-configuration-parameters_{context}"]
+= Installation configuration parameters for {rh-virtualization}
+
+Before you deploy an {product-title} cluster, you provide parameter values to describe your account on the cloud platform that hosts your cluster and optionally customize your cluster's platform. When you create the `install-config.yaml` installation configuration file, you provide values for the required parameters through the command line. If you customize your cluster, you can modify the `install-config.yaml` file to provide more details about the platform.
+
+The following example is specific to installing {product-title} on {rh-virtualization}. It uses numbered callouts to show which parameter values you can edit. Do not modify the parameters values without callouts.
+
+This file is located in the `<installation directory>` you specified when you ran the following command.
+----
+$ sudo ./openshift-install create install-config --dir=<installation_directory>
+----
+
+[NOTE]
+====
+* Do not copy the following example. Instead, run the installation program to create one.
+* You cannot modify these parameters in the `install-config.yaml` file after installation.
+====
+
+IMPORTANT: If you make customizations that require additional resources, such as adding control plane and compute machines, verify that your {rh-virtualization} environment has enough resources. Otherwise, these customizations might cause the installation to fail.
+
+
+.Example `install-config.yaml` configuration file
+
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: <virtlab.example.com> <1>
+compute:
+- hyperthreading: Enabled
+  name: worker
+  platform: {}
+  replicas: 3 <2>
+controlPlane:
+  hyperthreading: Enabled
+  name: master
+  platform: {}
+  replicas: 3 <3>
+metadata:
+  name: <my-cluster> <4>
+platform:
+  ovirt:
+    api_vip: <ip-address> <5>
+    dns_vip: <ip-address> <6>
+    ingress_vip: <ip-address> <7>
+    ovirt_cluster_id: <rhv-cluster-id> <8>
+    ovirt_storage_domain_id: <rhv-storage-domain-id> <9>
+publish: External
+pullSecret: |
+  <pull-secret> <10>
+sshKey: |
+  <ssh-public-key> <11>
+----
+<1> For `<virtlab.example.com>`, specify the base domain of the {product-title} cluster.
+<2> Specify `2` or more compute machines. The default value is `3`.
```

Comment:
Why we instruct users to specify 2 or more, if the default is 3?

Reply:
I think the valid range for IPI starts at 2. However, since we haven't tested it, let's start with 3.